auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -218,7 +232,6 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeString(mToken);
         dest.writeString(mSecret);
         dest.writeInt(mIsNewUser ? 1 : 0);
-
```

Comment:
Nit: leave the space here since this is an unpleasant, unconventional write below.

Reply:
Done.